cmd/ponzu/add.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// use `go get` to download addon and add to $GOPATH/src, useful
+// for IDE auto-import and code completion, then copy entire directory
+// tree to project's ./addons folder
+func getAddon(args []string) error {
+
+	var cmdOptions []string
+	var addonPath = args[1]
+
+	// Go get
+	cmdOptions = append(cmdOptions, "get", addonPath)
+	get := exec.Command(gocmd, cmdOptions...)
+	get.Stderr = os.Stderr
+	get.Stdout = os.Stdout
+
+	err := get.Start()
+	if err != nil {
+		return addError(err)
+	}
+	err = get.Wait()
+	if err != nil {
+		return addError(err)
+	}
+
+	// copy to ./addons folder
+	// GOPATH can be a list delimited by ":" on Linux or ";" on Windows
+	// `go get` uses the first, this should parse out the first whatever the OS
+	gopath := resolveGOPATH()
```

Comment:
if you don't mind, before we merge this in, could you take a look at #82 and see how it would impact GOPATH resolution in the way you have it? 

#82 hasn't been fixed, but the code is pretty much written in the issue... if you felt inclined to add that in I would be grateful.